src/styles/components/dropdown.scss

```diff
@@ -123,31 +140,78 @@ $toggle-text-color: var(--toggle-text-color);
 	}
 }
 
+// Input group style
 .input-group {
-	& .input-group-text {
-		.dropdown-toggle.btn-secondary {
+	& .input-group-text, & .input-group-prepend {
```

Comment:
@Aringoaway I think the same style should be implemented for `input-group-append`